assets/charts/control-plane/kube-apiserver/templates/kube-apiserver-secret.yaml

```
@@ -2,6 +2,7 @@
   apiserver.key: "{{ .Values.apiserver.apiserverKey }}"
   apiserver.crt: "{{ .Values.apiserver.apiserverCert }}"
   service-account.pub: "{{ .Values.apiserver.serviceAccountPub }}"
+  service-account.private: "{{ .Values.apiserver.serviceAccountPrivate }}"
```

Usually private keys has `key` extension.
```suggestion
  service-account.key: "{{ .Values.apiserver.serviceAccountPrivate }}"
```